tmo-monitor.py

```diff
@@ -31,9 +33,14 @@ def login_app(self):
     self.app_jar.set('lsid', login_request.cookies['lsid'], domain='192.168.12.1', path='/')
 
   def get_site_info(self):
-    if not self.app_jar:
-      self.login_app()
-    stat_request = requests.get('http://192.168.12.1/cell_status_app.cgi', cookies=self.app_jar)
+    try:
+      if not self.app_jar:
+        self.login_app()
+      stat_request = requests.get('http://192.168.12.1/cell_status_app.cgi', cookies=self.app_jar)
+    except:
+      print ("Could not query site info, exiting.")
```

Comment:
Calls to `print` are inconsistent in style throughout the script between `print(` and `print (`